contracts/icp/context-config/src/query.rs

```diff
@@ -134,3 +134,14 @@ fn privileges(
         privileges
     })
 }
+
+#[ic_cdk::query]
+fn fetch_nonce(context_id: ICRepr<ContextId>, member_id: ICRepr<ContextIdentity>) -> Option<u64> {
+    with_state(|configs| {
+        configs
+            .contexts
+            .get(&context_id)
+            .and_then(|context| context.member_nonces.get(&member_id))
+            .copied()
```

Comment:
for consistency, we should probably panic if the context does not exist

same as we have it above

Reply:
but should we be panicking in ICP at all? how do they handle that?